style.css

```diff
@@ -11,16 +13,15 @@ html{
 .content{
 	background-color: #fbfcf8;
 	display: grid;
-	grid-template-rows: 75px 200px 550px 750px 50px;

```

Comment:
I changed the fixed heights of the websites row to 'auto' to allow the site to be more responsive